src/internal/connector/graph_connector.go

```diff
@@ -237,6 +239,12 @@ func identifySite(item any) (string, string, error) {
 		return "", "", errors.Errorf("no name for Site: %s", *m.GetId())
 	}
 
+	// personal (ie: oneDrive) sites have to be filtered out server-side.
+	url := m.GetWebUrl()
+	if url != nil && strings.Contains(*url, personalSitePath) {
```

Comment:
I'm not familiar with how sharepoint site URLs are formatted, but is it possible for users to generate a site with a URL that would have `sharepoint.com/personal` in it but not be a personal site?

Reply:
Looks like the slashes get stripped from the generated url.

<img width="506" alt="image" src="https://user-images.githubusercontent.com/104464746/203421706-e2aaf61d-400c-4a80-b033-22a3a049d738.png">